test/functional/passwords_controller_test.rb

```diff
@@ -127,7 +127,7 @@ class PasswordsControllerTest < ActionController::TestCase
         put :update, params: {
           user_id: @user.id,
           token: @user.confirmation_token,
-          password_reset: { password: "password1234" }
+          password_reset: { password: "@61:details:THOSE:present:71@" }
```

Comment:
Any reason to not use `SECURE_TEST_PASSWORD` here as well (and in all similar cases)?

Reply:
None, I just forgot that I'd used a couple of different test passwords…